packages/engine-server/src/config/configTemplate.ts

```diff
@@ -7,6 +7,14 @@ export const configTemplate: any = {
 			password: `%tenant.env.DB_PASSWORD%`,
 			database: `%tenant.env.DB_NAME%`,
 			ssl: `%?tenant.env.DB_SSL::bool%`,
+			max: '%?tenant.env.DB_POOL_MAX%',
```

Comment:
* nemělo by to všechno být pod klíče `pool`?
* z názvu max není jasný max čeho to je, tj. lepší maxTotal nebo maxConnections